latex/ledger-spec.tex

```diff
@@ -277,6 +394,16 @@ \section{UTxO}
     & \fun{balance} \in \UTxO \to \Coin
     & \text{UTxO balance} \\
     & \fun{balance} ~ utxo = \sum_{(~\wcard ~ \mapsto (\wcard, ~c)) \in \var{utxo}} c
+    \nextdef
+    & \fun{created} \in \PrtclConsts \to \UTxO \to \Allocs \to \Allocs \to \Tx \to \Coin
+    & \text{value created} \\
+    & \created{pc}{utxo}{dallocs}{pallocs}{tx} = \\
+    & ~~\balance{(\txins{tx} \restrictdom \var{utxo})} + \refunds{pc}{dallocs}{pallocs}{tx}
+    \nextdef
+    & \fun{destroyed} \in \PrtclConsts \to \Tx \to \Coin
```

Comment:
is `destroyed` a good name for this ?

Reply:
I'm open to suggestions!  I do think of those UTxO as being destroyed and new ones being created, and the other things are values which guide the calculation.

Reply:
ah, for UTxO it' ok. I had somehow associated this with funds